### DISCUSSION TITLE
Add support for Default Upload folder for M2M Images/Files

### DISCUSSION THREAD

**Discussion by abertay-university** [directus/directus#7143] [Feature Requests]
### Preflight Checklist

- [X] I have completed all [Troubleshooting Steps](https://docs.directus.io/getting-started/support/#troubleshooting-steps).
- [X] I'm on [the latest version of Directus](https://github.com/directus/directus/releases).
- [X] There's [no other issue](https://github.com/directus/directus/issues) that already describes my problem.

### Describe the Bug

When creating a M2O field, I can choose what folder any new uploads are saved. However, this is not present for Multiple Files Field "M2M".

### To Reproduce

1. Create a new Single File field
2. Note the Upload Folder option
3. Create a new Multiple Files field
4. Note the Upload folder option is not there

**Single File Field**
![image](https://user-images.githubusercontent.com/31690037/127873730-a7e26f33-dc21-4eb1-9922-865707babd8a.png)

**Multiple Files field**
![image](https://user-images.githubusercontent.com/31690037/127873902-6e800876-07f3-4ff0-b288-55b67e87cb9e.png)

Maybe something like this will be good:
![image](https://user-images.githubusercontent.com/31690037/127874296-35c6e06b-b811-4e3b-a314-7bdcf1d233b4.png)
![image](https://user-images.githubusercontent.com/31690037/127874193-41e7c403-7c3c-4f77-88b8-b432f7dc8209.png)

Having a separate interface for M2M images also allows for a card/list view

### What version of Directus are you using?

9.0.0r87

### What version of Node.js are you using?

14

### What database are you using?

MySQL

### What browser are you using?

Chrome, Safari, Firefox

### What operating system are you using?

MacOS

### How are you deploying Directus?

locally

**Comment by LasseRosenow** [comment]
I just realized that this seems to be fixed in the current directus version:

![image](https://user-images.githubusercontent.com/10547444/148650439-5169c7f7-6b23-42cb-9de4-02760b56dd92.png)

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
